src/app/components/all-products-components/all-products-content/all-products-content.component.ts

```diff
@@ -9,13 +11,57 @@ import {ProductCard} from "../../../interfaces/product-card";
   encapsulation: ViewEncapsulation.None
 })
 export class AllProductsContentComponent implements OnInit {
+
   @Input() products: ProductCard[] = [];
-  @Input() productsQuantity?:number;
-  p: number = 3;
+  @Input() productsQuantity: number;
+  public p: number = 3;
+  public itemsPerPage: number = 5;
+  public farmValue: string[] = [];
+  public filteredItems: ProductCard[] = [];
+
 
-  constructor() { }
+  constructor(private filtersService: FiltersService) {}
 
   ngOnInit(): void {
+    this.sendItemsPerPage();
+  }
+
+
+  sendItemsPerPage(): void {
```

Comment:
Please add access modifier for each method (public, private or protected)

Reply:
+Fixed accessibility in all components.